corporate/address-changes.rst

```diff
@@ -1,31 +1,127 @@
+
+Update the owner, administrative contact, technical contact, and billing contact for all registered domains.
+
+Google Voice
+------------
+
+Update the :guilabel:`Organization Address` in the `Google Admin Console <https://admin.google.com>`_, under :menuselection:`Billing --> Subscriptions --> Google Voice Starter --> View payment settings`.
+
+Google Play Developer
+---------------------
+
+After updating the address with :ref:`Dun & Bradstreet`, Google Play will send a notification to the account owner.
+
+.. seealso::
+   See the `Google Play Developer support documentation <https://support.google.com/googleplay/android-developer/answer/13634888>`_ for more information.
+
+Apple Developer
+---------------
+
+Update :guilabel:`Membership details` in `Apple Developer <https://developer.apple.com/account>`_.
+
+Apple uses :ref:`Dun & Bradstreet` for business verification, so the address should be updated there first.
+
+Georgia Tech vendor registration
+--------------------------------
+
+Submit a `ServiceNow request <https://gatech.service-now.com/guest?id=sc_cat_item&sys_id=9ade0cf4db81c010391b9837db961948&sysparm_category=9da900ebdbb08858391b9837db9619af>`_.
+
+.. note::
+   The vendor registration is currently unused, as all payments from Georgia Tech are sent via the :ref:`external committee member <Georgia Tech external committee member>`.
+
+Georgia Tech external committee member
+--------------------------------------
+
+This is manually managed by finance professionals at the Institute. Notify the student treasurer of the address change.
+
+Bitwarden
+---------
+
+Update the :guilabel:`Tax information` section under :menuselection:`Admin Console --> Billing --> Payment method` in the `Bitwarden Web Vault <https://vault.bitwarden.com>`_.
+
+Postmark
+--------
+
+Update the billing address for the card on file under :menuselection:`Account --> Billing Details` in `account settings <https://account.postmarkapp.com/billing_settings>`_.
+
+Northwest Registered Agent
+--------------------------
+
+Update the billing address for the card on file during the next payment.
+
+Corporation website
+-------------------
 
-The following list of places have the RoboJackets, Inc. address on file and need to be updated if the address changes:
-
-* secretary of state
-* Domain name WHOIS
-* mercury
-* BILL spend & expense
-* Gandi
-* corporate website
-* Internal Revenue Service - https://www.irs.gov/charities-non-profits/other-non-profits/change-of-address-exempt-organizations
-* gt vendor registration
-* gt external committee member for check payments
-* dun and Bradstreet
-* Georgia department of revenue
-* google play
-* probably apple developer account if/when that becomes a thing
-* square
-* Google Voice - in the Google Admin console
-* Bitwarden billing
-* northwest billing
-* postmark billing
+Update `this Markdown file <https://github.com/RoboJackets/corporate.robojackets.org/blob/gh-pages/index.md>`_.
```

Comment:
Is it on robojackets.org/contact? I'd check, but...

Reply:
I don't believe so, it should link out to https://corporate.robojackets.org. I'll confirm once we're back up.